resources/qgis-resource-metadata.xsd

```diff
@@ -22,14 +22,14 @@
 ###############################################################################
 -->
 <xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:rm="http://qgis.org/resource-metadata/1.0" targetNamespace="http://qgis.org/resource-metadata/1.0" elementFormDefault="qualified" attributeFormDefault="unqualified" version="1.0">
-	<xs:element name="ResourceMetadata" type="rm:ResourceMetadataType">
+	<xs:element name="resourceMetadata" type="rm:resourceMetadataType">
```

Comment:
What's the rationale on this change? Just curious.

Reply:
I found more XML tags in the QGIS project file starting with a small letter, so I thought we might want uniform them. I can of course rollback, no strong opinion.